tests/lib/rules/require-meta-docs-recommended.js

```diff
@@ -0,0 +1,178 @@
+'use strict';
+
+const rule = require('../../../lib/rules/require-meta-docs-recommended');
+const RuleTester = require('../eslint-rule-tester').RuleTester;
+
+const ruleTester = new RuleTester({
+  languageOptions: { sourceType: 'commonjs' },
+});
+
+ruleTester.run('require-meta-docs-recommended', rule, {
+  valid: [
```

Comment:
Can you make sure we allow situations where we can't statically determine the value? Sometimes people may use functions or other dynamic values that we can't analyze, and we generally want to ensure that's allowed. We shouldn't only flag a problem when we detect an actual invalid value or the property is obviously missing.

```js
module.exports = {
        meta: { docs: { recommended: RECOMMENDED } },
        create(context) {}
      };
```

Reply:
Added. I'll note that this and existing similar functions don't handle the case of `meta: (unknown)` or `meta: { docs: (unknown) }` well.